README.rst

```diff
@@ -2,14 +2,17 @@
 Session Video Uploader
 ======================
 
-Upload session videos to YouTube.
+Support features:
+- Upload session videos to YouTube.
+- Generate PyCon YouTube channel video data.
```

Comment:
I think we could add the purposes for why we should support generating PyCon YouTube video info. E.g. "Generating PyCon YouTube channel video data for PyVideo usage".

Reply:
Done